### ISSUE TITLE
[FEATURE] Add cost to respawn with gear

### ISSUE THREAD

**Issue by strideynet** [LexTheGreat/dnicetlx.Takistan#20]
Add cost for blufor/opfor each time they respawn with their gear. Perhaps 50% of gear value. Encourages preservation of life.

**Comment by strideynet** [comment]
@LexTheGreat ?

**Comment by GiovanniPorta** [comment]
I don't agree with this because not everyone wants to keep their gear when respawning, therefore charging them 50℅ shouldn't be necessary. Say someone goes and buys that one expensive launcher that they need to use to kill a tank - they should not be charged for its value when they die.

**Comment by strideynet** [comment]
However we could easily write a dialogue to allow people to opt in or opt out upon each death
